src/ICeWall/test/tcl_interface.flipchip.example.tcl

```diff
@@ -0,0 +1,422 @@
+add_pad -edge left   -inst_name u_vzz_31 -signal DVSS_0 -type vssio -location {origin {x   35 y 1260}} -bump {row 11 col 4}
+add_pad -edge left   -inst_name u_v18_31 -signal DVDD_0 -type vddio -location {origin {x   35 y 1230}} -bump {row 11 col 2}
+add_pad -edge left   -signal p_ddr_dqs_p_0_io           -type sig   -location {origin {x   35 y 1200}} -bump {row 11 col 1}
+add_pad -edge left   -signal p_ddr_dq_15_io             -type sig   -location {origin {x   35 y 1170}} -bump {row 11 col 3}
+add_pad -edge left   -inst_name u_cbrk17                -type cbk   -location {origin {x   35 y 1140}}
+add_pad -edge left   -signal p_ddr_dq_14_io             -type sig   -location {origin {x   35 y 1135}} -bump {row 11 col 5}
+add_pad -edge left   -signal p_ddr_dq_13_io             -type sig   -location {origin {x   35 y 1095}} -bump {row 12 col 4}
+add_pad -edge left   -inst_name u_vzz_32 -signal DVSS_0 -type vssio -location {origin {x   35 y 1065}} -bump {row 12 col 2}
+add_pad -edge left   -inst_name u_v18_32 -signal DVDD_0 -type vddio -location {origin {x   35 y 1035}} -bump {row 12 col 1}
+add_pad -edge left   -signal p_ddr_dq_12_io             -type sig   -location {origin {x   35 y 1005}} -bump {row 12 col 3}
+add_pad -edge left   -signal p_ddr_dq_11_io             -type sig   -location {origin {x   35 y  975}} -bump {row 12 col 5}
+add_pad -edge left   -inst_name u_pwrdet17              -type pdt   -location {origin {x   35 y  940}}
+add_pad -edge left   -signal p_ddr_dq_10_io             -type sig   -location {origin {x   35 y  935}} -bump {row 13 col 4}
+add_pad -edge left   -signal p_ddr_dq_9_io              -type sig   -location {origin {x   35 y  905}} -bump {row 13 col 2}
+add_pad -edge left   -signal p_ddr_dq_8_io              -type sig   -location {origin {x   35 y  875}} -bump {row 13 col 1}
+add_pad -edge left   -inst_name u_vdd_15 -signal VDD    -type vdd   -location {origin {x   35 y  845}} -bump {row 13 col 3}
+add_pad -edge left   -inst_name u_vss_15 -signal VSS    -type vss   -location {origin {x   35 y  745}} -bump {row 14 col 2}
+add_pad -edge left   -inst_name u_vzz_33 -signal DVSS_0 -type vssio -location {origin {x   35 y  715}} -bump {row 14 col 1}
+add_pad -edge left   -inst_name u_v18_33 -signal DVDD_0 -type vddio -location {origin {x   35 y  685}} -bump {row 14 col 3}
+add_pad -edge left   -signal UNASSIGNED                 -type sig   -location {origin {x   35 y  555}} -bump {row 15 col 2}
+add_pad -edge left   -signal UNASSIGNED                 -type sig   -location {origin {x   35 y  525}} -bump {row 15 col 1}
+add_pad -edge left   -signal UNASSIGNED                 -type sig   -location {origin {x   35 y  395}} -bump {row 16 col 1}
+
+if {[catch {initialize_padring} msg]} {
+  puts $errorInfo
+  puts $msg
+  return
+}
+
+set def_file [make_result_file "tcl_interface.flipchip.example.def"]
+
+write_def $def_file
+diff_files $def_file "tcl_interface.flipchip.defok"
+
```

Comment:
You probably will want to include the example of merging the def files here until it's been fixed.

Reply:
Example added